app/src/androidTest/java/com/google/jetpackcamera/utils/AppTestUtil.kt

```diff
@@ -28,7 +28,7 @@ import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.callbackFlow
 import kotlinx.coroutines.flow.transform
 
-const val TAG = "AppTestUtil"
+const val APP_TEST_UTIL_TAG = "AppTestUtil"
```

Comment:
Instead of renaming this, you should just make it `private const val TAG = "AppTestUtil"`